portsyncd/portsyncd.cpp

```diff
@@ -69,7 +71,7 @@ int main(int argc, char **argv)
     DBConnector cfgDb(CONFIG_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
     DBConnector appl_db(APPL_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
     DBConnector state_db(STATE_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
-    ProducerStateTable p(&appl_db, APP_PORT_TABLE_NAME);
```

Comment:
Could you do the ProducerStateTable  to ProducerTable  change in a separate PR?  It is not strictly related to warm reboot.

Reply:
Let's start another PR for it. Please help [https://github.com/Azure/sonic-swss/pull/556](https://github.com/Azure/sonic-swss/pull/556)

---
In reply to: [205866896](https://github.com/Azure/sonic-swss/pull/551#discussion_r205866896) [](ancestors = 205866896)

Reply:
Your suggestion is good. Let's keep ProducerStateTable in this PR.

---
In reply to: [206734165](https://github.com/Azure/sonic-swss/pull/551#discussion_r206734165) [](ancestors = 206734165,205866896)